tests/unit/test_merger.py

```diff
@@ -272,3 +272,55 @@ def test_real_record_merge_regression_1_authors_mismatch_on_update():
     assert sorted(conflict, key=itemgetter("path")) == sorted(
         expected_conflicts, key=itemgetter("path")
     )
+
+
+def test_titles_change():
```

Comment:
maybe split this into multiple tests (or parametrize it) for clarity?

Reply:
:+1: